shared_kv_storage/Dockerfile

```diff
@@ -42,26 +41,48 @@ RUN apt-get update \
 
 RUN pip3 install --upgrade setuptools
 
-ENV TCF_HOME=/project/avalon
-
 # Make Python3 default
 RUN ln -sf /usr/bin/python3 /usr/bin/python
 
-COPY . /project/avalon/
+
+# Build Shared KV intermediate docker image.
+FROM base_image as build_image

```

Comment:
Delete the above "FROM base_image as build_image"

Reply:
This is done in order to avoid including source code (eg: cpp files) in final image. We have base_image, build_image and final_image.  Final image is built based on base_image which has packages installed required to run shared_kv . Final image only copies the required build artifacts from build_image. If I remove base_image and only have build_image then final image has to be built based on build_image which will contain the source code and selectively copying required packages from build_images to run binary becomes non trivial.

Reply:
Do you have the image (avalon-lmdb-dev) size difference with this change and without this change?

Reply:
I didn't check. I only verified that the docker container contains only required files and doesn't have files that are not required for shared kv for example contents of tc folder. But I think there should be some more room for optimizing the docker image size if we if install some packages in build_image instead of base_image

Reply:
Size reduced by 70%. 530 MB to 159 MB.